rust/index/src/hnsw.rs

```diff
@@ -3,15 +3,16 @@ use chroma_error::{ChromaError, ErrorCodes};
 use chroma_types::{Metadata, MetadataValue, MetadataValueConversionError, Segment};
 use std::ffi::CString;
 use std::ffi::{c_char, c_int};
+use std::path::Path;
 use std::str::Utf8Error;
 use thiserror::Error;
 use tracing::instrument;
 use uuid::Uuid;
 
-const DEFAULT_MAX_ELEMENTS: usize = 10000;
-const DEFAULT_HNSW_M: usize = 16;
-const DEFAULT_HNSW_EF_CONSTRUCTION: usize = 100;
-const DEFAULT_HNSW_EF_SEARCH: usize = 10;
+pub const DEFAULT_MAX_ELEMENTS: usize = 10000;
```

Comment:
why is this blanket pub

Reply:
Because these params are used by the hnsw segment which is a different crate. Alternatively, I could move these there but I feel like the index type is more apt for these